### ISSUE TITLE
NA values in phenotype/.fam file

### ISSUE THREAD

**Issue by mbytc1** [precimed/mostest#9]
If possible, please could you provide a fix so that is is possible to provide NA values for participants/samples across the phenotype file (e.g. if have removed due to being relatedness or population stratification)? (NB i have made sure that they are consistent participants between phenotype and bfiles). Currently, if I try to run it with NA values for any participant, the output all looks correct aside from the univariate sumstats output, where no BETA and SE values are provided.  I know that I could instead run an extra stage before MOStest of making new bfiles using plink with limiting to those participants with no NA values, but if possible, it would be great and much simpler if there is a fix within MOSTest to allow it. Thank you

**Comment by zqsha** [comment]
Yes, completely agree. I met the same issue. Hopefully, the developers can fix this issue. Or already updated?

**Comment by denvdm** [comment]
Hi, sorry for late reply. While this generated an (unnecessary) error with the earliest versions of MOSTest, it should run smoothly now, if you set missing values to 'NaN'.
